apps/prairielearn/src/migrations/20240111004522_group_users__group_config_id__not_null.sql

```diff
@@ -0,0 +1,28 @@
+-- After backfilling, we still had some `group_users` rows where `group_config_id` was NULL.
+-- This is because some `groups` had a NULL `group_config_id`. We'll be fixing that separately,
+-- but we'll remove those invalid rows from `group_users` so we can add the constraint.
+DELETE FROM group_users WHERE group_config_id IS NULL;
```

Comment:
This should be safe: I'm pretty confident that `groups` rows without a `group_config_id` are unreachable via all the joins we use to query them, and thus all `group_users` for those groups are as well.